oap-logstream/src/main/java/oap/logstream/disk/DiskLoggerBackend.java

```diff
@@ -70,6 +71,7 @@ public class DiskLoggerBackend extends AbstractLoggerBackend implements Cloneabl
     private final LoadingCache<LogId, AbstractWriter<? extends Closeable>> writers;
     private final ScheduledExecutorService pool;
     public String filePattern = "/${YEAR}-${MONTH}/${DAY}/${LOG_TYPE}_v${LOG_VERSION}_${CLIENT_HOST}-${YEAR}-${MONTH}-${DAY}-${HOUR}-${INTERVAL}.tsv.gz";
```

Comment:
Why `.tsv.gz` is hardcoded for `DiskLoggerBackend`?
Possibly extension should not be present in `filePattern`?

Reply:
Got it offline. `filePattern` is specified via OAP config